### DISCUSSION TITLE
Ruby? - I use PHP

### DISCUSSION THREAD

**Discussion by ErikThiart** [tabler/tabler#1424] [Q&A] [answered]
The documentation says the following:

Set up the environment
To use our build system and run our documentation locally, you’ll need a copy of Tabler’s source files. Follow the steps below:

[Install Node.js](https://nodejs.org/download/), which we use to manage our dependencies.
Navigate to the root /tabler directory and run npm install to install our local dependencies listed in package.json.
[Install Ruby](https://ruby-lang.org/en/documentation/installation/) - the recommended version is [2.5.5](https://cache.ruby-lang.org/pub/ruby/2.5/ruby-2.5.5.tar.gz).
[Install Bundler](https://bundler.io/) with gem install bundler and, finally, run bundle install. It will install all Ruby dependencies, such as [Jekyll and plugins](https://jekyllrb.com/).

--

Is there a way I can just use this as a normal bootstrap project with vanilla PHP, no framework and funny business, I just want the CDN stylesheets, Javascript and Charts

Then use PHP for the rest?

Or is it a must to use NodeJS etc?

I host on Cpanel which supports PHP and MySQL only.

**Comment by martyncoup** [comment, marked as answer]
Yes, I use it in .NET MVC and Angular with no issues. The stylesheets are not on a CDN I don't believe, but the JS is (also discussed in the docs). If you wanted to edit the SCSS, then you can use Gulp to compile and minify your stylesheets.

The distributed copy in the download is just static HTML so you can use it however you wish with whatever backend you like.

**Comment by ErikThiart** [comment]
Perfect, so I can download the normal CSS and work from there?

**Comment by martyncoup** [comment]
Yes, you can find it in the dist folder.

From: Erik Thiart ***@***.***>
Date: Friday, 6 January 2023 at 16:00
To: tabler/tabler ***@***.***>
Cc: Martyn Coupland ***@***.***>, Comment ***@***.***>
Subject: Re: [tabler/tabler] Ruby? - I use PHP (Discussion #1424)

Perfect, so I can download the normal CSS and work from there?
